src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -181,6 +192,28 @@
     block_public_policy=True,
 )
 
+######################
+# Manage Consul Data #
+######################
+consul_kv_data = {
+    "s3-storage-bucket": storage_bucket_name,
+    "s3-grades-bucket": grades_bucket_name,
+    "s3-course-bucket": course_bucket_name,
+    "lms-domain": edxapp_domains["lms"],
+    "studio-domain": edxapp_domains["studio"],
+    "preview-domain": edxapp_domains["preview"],
+    "cookie-domain": edxapp_zone["domain"].apply(".{}".format),
+    "ses-mail-domain": edxapp_mail_domain,
```

Comment:
Should this be part of the `edxapp_domains` dict?

Reply:
Possibly?